content/artifacts/artifactory/plugin.js

```diff
@@ -326,15 +326,46 @@ function Untaken({ selected }) {
   }
 
   let planetList = {
-    maxHeight: '100px',
+    maxHeight: '300px',
     overflowX: 'hidden',
     overflowY: 'scroll',
   };
+  const inputGroup = {
+    display: 'flex',
+    alignItems: 'center',
+  };
+  const input = {
+    flex: '1',
+    padding: '5px',
+    margin: 'auto 5px',
+    outline: 'none',
+    color: 'black',
+  };
+
+  let {x: homeX, y: homeY} = ui.getHomeCoords()
 
   let [lastLocationId, setLastLocationId] = useState(null);
+  let [centerX, setCenterX] = useState(homeX);
+  let [centerY, setCenterY] = useState(homeY);
+  
+  const onChangeX = (e) => {
+    return setCenterX(e.target.value)
+  }
+
+  const onChangeY = (e) => {
+    setCenterY(e.target.value)
+  }
 
   const planets = allPlanetsWithArtifacts()
     .filter(isUnowned);
+  
+  planets.forEach(planet => {
+    let x = planet.location.coords.x;
+    let y = planet.location.coords.y;
+    planet.location.distanceFromHome = parseInt(Math.sqrt(Math.pow((x-centerX),2) + Math.pow((y-centerY),2)));
```

Comment:
Can we keep this data contained to the module instead of attaching it to the game's planet object? Can probably use a Map keyed by the locationId

Reply:
gonna just make a `planetsArray` with `{locationId, x, y, distanceFromTargeting}` in it, and use that to populate the UI instead of `planets`